### PR TITLE
Added ARG query for "Use Azure Arc-enabled servers to control software updates deployments to servers" in azure_arc_checklist.en.json 

### PR DESCRIPTION
**Pull request by denver-reed**
## Description
Please provide a detailed description of your changes.
- Updated description for Automanage recommendations to instead point to Azure Machine Configuration
- Updated wording for managing updates to use Azure Update Manager to schedule updates
- Included ARG query for checking if Arc machines are attached to a maintenance schedule

## Related Issue
Link to any related issues or discussions here. This helps reviewers understand the context and the need for your changes.

## Checklist

- [x] I've tested my changes to ensure they are ready for review.
- [x] I've read the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) guide.
- [ ] I've updated the documentation (if applicable).
- [x] Resource Graph queries have been included (and tested) for recommendations where ever possible[^note].
- [ ] Resource Graph queries have NOT been included (please explain below).

## Additional Information
Is there any additional context, screenshots, or considerations that might help in the review process? Please include them here.

## Reviewer Notes
Is there a specific area you’d like feedback on? Please highlight it here. We're here to help and learn together! 💡

[^note]:
    Details on how to add Azure Resource Graph queries to recommendations can be found [here](../blob/main/CONTRIBUTING.md#1-adding-or-modifying-resource-graph-queries).

### CODE REVIEW

**Comment by jodyford-msft** [comment]
Please add line item ID and checklist filename to the title of your PR

**Comment by denver-reed** [comment]
> Please add line item ID and checklist filename to the title of your PR

Confirmed file name is in the title. Unfortunately, for this checklist, there are no line item IDs.

File: checklists/azure_arc_checklist.en.json [hunk 1]
```diff
@@ -190,11 +190,11 @@
...
             "subcategory": "Management",
             "waf": "Operations",
-            "text": "Consider using Azure Automanage to control settings and avoid configuration drift on servers",
-            "description": "Azure Automanage help implement Microsoft best-practices for servers management in Azure",
+            "text": "Consider using Azure Machine Configuration to control settings and avoid configuration drift on servers",
+            "description": "Azure Machine Configuration to help implement Microsoft best-practices for servers management in Azure",
             "guid": "7a927c39-74d1-4102-aac6-aae01e6a84de",
             "severity": "Medium",
-            "link": "https://learn.microsoft.com/azure/automanage/automanage-arc"
+            "link": "https://learn.microsoft.com/en-us/azure/governance/machine-configuration/overview"
```
Comment (erjosito) [inline checklists/azure_arc_checklist.en.json:197]: Hey could you please remove the localization (`en-us/`) from the link?

File: checklists/azure_arc_checklist.en.json [hunk 2]
```diff
@@ -246,10 +246,11 @@
...
+            "description": "Use Azure Update Manager to manage and schedule updates at scale across your Azure Arc-enabled servers",
             "guid": "ae2cc84c-37b6-4b78-8cba-fe6c46589d45",
             "severity": "Low",
-            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/manage/hybrid/server/best-practices/arc-update-management"
+            "graph": "resources | where type =~ 'microsoft.hybridcompute/machines' | project id = tolower(id), arcMachineName = name | join kind=leftouter (maintenanceresources | extend baseIdParts = split(tolower(id), '/providers/microsoft.maintenance/') | extend maintenanceMachineId = tostring(baseIdParts[0]) | project maintenanceMachineId, maintenanceId = id) on $left.id == $right.maintenanceMachineId | extend compliant = iif(maintenanceMachineId == '', 'No', 'Yes') | project id, compliant",
+            "link": "https://learn.microsoft.com/en-us/azure/update-manager/scheduled-patching?tabs=schedule-updates-single-machine%2Cschedule-updates-scale-overview%2Cwindows-maintenance"
```
Comment (erjosito) [inline checklists/azure_arc_checklist.en.json:253]: Hey could you please remove the localization (`en-us/`) from the link? Thanks!

**Review by erjosito — CHANGES_REQUESTED** [review]

File: checklists/azure_arc_checklist.en.json (hunk 1 above)
Comment (denver-reed) [reply checklists/azure_arc_checklist.en.json]: Resolved!

File: checklists/azure_arc_checklist.en.json (hunk 2 above)
Comment (denver-reed) [reply checklists/azure_arc_checklist.en.json]: Resolved!

**Review by erjosito — APPROVED** [review]

**Pull request merged**